src/metabase/util/i18n/impl.clj

```diff
@@ -31,6 +31,15 @@
           (str language \_ (some-> country str/upper-case))
           language)))))
 
+(defn migrate-legacy-locale-string
```

Comment:
```suggestion
(defn- migrate-legacy-locale-string
```

Let's make this private to make it clear that it's not used anywhere outside of this namespace... if we want to refactor it down the road it will be easier to do because we won't have to grep the codebase for usages